frontend/packages/kubevirt-plugin/src/constants/vm/storage/access-mode.ts

```diff
@@ -2,9 +2,16 @@
 import { ObjectEnum } from '../../object-enum';
 
 export class AccessMode extends ObjectEnum<string> {
-  static readonly SINGLE_USER = new AccessMode('ReadWriteOnce');
-  static readonly SHARED_ACCESS = new AccessMode('ReadWriteMany');
-  static readonly READ_ONLY = new AccessMode('ReadOnlyMany');
+  static readonly READ_WRITE_ONCE = new AccessMode('ReadWriteOnce', 'Single User (RWO)');
+  static readonly READ_WRITE_MANY = new AccessMode('ReadWriteMany', 'Shared Access (RWX)');
+  static readonly READ_ONLY_MANY = new AccessMode('ReadOnlyMany', 'Read Only (ROX)');
```

Comment:
IMO we should use the k8s names as it is can be confusing for a programmer to map these to the k8s ones

@pcbailey